components/remote-environment-broker/internal/broker/provision_test.go

```diff
@@ -189,7 +191,7 @@ func TestProvisionWhenProvisioningInProgress(t *testing.T) {
 	assert.Equal(t, &expOpKey, actResp.OperationKey)
 }
 
-func TestProvisionErrorOnCreatingEventActivation(t *testing.T) {
+func TestProvisionCreatingEventActivation(t *testing.T) {
 	// GIVEN
```

Comment:
all `given` statements should be inside the `t.Run` function

you cannot reuse assertion between different test cases